app/models/EORI.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+case class EORI(value: String)
```

Comment:
:+1:

Reply:
Please bear in mind potential problems with Mockito: https://gist.github.com/kasia-kittel/3255fd5dc5f8f861f4e02bb966302c1c
You might need to use `anyVal[T]` from `mockito-scala`: https://github.com/mockito/mockito-scala

Besides, I've recently changed that not to extend `AnyVal` because of the problems with some tests. I don't think there is much to gain to be fair.